tutorials/4_RunningREMINDandMAgPIE.md

```diff
@@ -64,7 +66,23 @@ See comments in the head section of the file. Most importantly you need to provi
 ### Configure the scenario_config_coupled.csv of your choice
 
 By default these are (A) `scenario_config_coupled_SSPSDP.csv` and (B) `scenario_config_SSPSDP.csv`. A provides some extra information
-for coupled runs (e.g. which run should be started). All other settings are taken from B. Thus every scenario in A must also be present in B.
+for coupled runs (e.g. which run should be started, specific MAgPIE configurations), with one scenario per row and settings on the columns. B is a normal `scenario_config` that defines REMIND settings, as explained in [3_RunningBundleOfRuns](https://github.com/remindmodel/remind/blob/develop/tutorials/4_RunningREMINDandMAgPIE.md). All other settings are taken from B. Thus every scenario in A must also be present in B.
+
+All the columns must be present in the `scenario_config_coupled.csv` file, but most of them can be left blank. The required ones are:
+   - `scenario`: The name of the scenario, must be unique and match the `scenario` column in REMIND's `scenario_config.csv`
+   - `start`: Defines if a scenario run should be started (1) or not (0). Overrides whatever is set in REMIND's `scenario_config.csv`. If you have an unfinished coupled run, it will automatically try to continue from the last coupling iteration (i.e. REMIND or MAgPIE run). If it already finished (i.e. reached `max_iterations`), it will simply try to restart the last REMIND run.
```

Comment:
I think it is not overwriting, but rather only running models with both start flags set to 1, see https://github.com/remindmodel/remind/blob/develop/start_bundle_coupled.R#L96

Reply:
This line keeps scenarios only that exist in both (standalone and coupled) scenario_config files. It does not care about the start setting. This is done by this line https://github.com/remindmodel/remind/blob/fe9bf59480ad406c351e12b10eedbee17f237f41/start_bundle_coupled.R#L82. As a results, `start` in `scenario_config_coupled.R` indeed overrides the `scenario_config.csv`.

Reply:
thanks!